modules/efk-logging-fluentd-limits.adoc

```diff
@@ -39,3 +39,27 @@ spec:
             memory: 1Gi
 ----
 <1> Specify the CPU and memory limits as needed. The values shown are the default values.
+
+----
+$ oc edit ClusterLogging instance
+
+apiVersion: "logging.openshift.io/v1"
+kind: "ClusterLogging"
+metadata:
+  name: "instance"
+
+....
+
+spec:
+  collection:
+    logs:
+      rsyslog:
+        resources:
+          limits: <1>
+            cpu: 250m
+            memory: 1Gi
+          requests:
+            cpu: 250m
+            memory: 1Gi
+----
```

Comment:
@mburke5678  I checked the default values of rsyslog's resources, they are:
    name: rsyslog
    resources:
      limits:
        memory: 358Mi
      requests:
        cpu: 100m
        memory: 358Mi

    name: logrotate
    resources:
      limits:
        memory: 358Mi
      requests:
        cpu: 100m
        memory: 358Mi